motionplan/solverFrame.go

```diff
@@ -148,21 +149,22 @@ func newSolverFrame(fs frame.FrameSystem, solveFrameName, goalFrameName string,
 	}
 
 	var ptgs []tpspace.PTGSolver
-	anyPTG := false     // Whether PTG frames have been observed
-	anyNonzero := false // Whether non-PTG frames
+	anyPTG := false // Whether PTG frames have been observed
```

Comment:
I needed to comment this out since we have introduced an `executionFrame` into the mix for solving...
Not immediately sure how to circumvent this.

Reply:
bump looking for advice on this

Reply:
We may want to move
```
	// re-root the frame system on the relative frame
	relativeOnlyFS, err := pm.frame.fss.FrameSystemSubset(request.Frame)
```
upstream of this so that this is a non-issue.

Reply:
hmmmmm, this is quite tricky...
I will attempt to describe why I think this is tricky best I can. First we describe our nomenclature.
Let `AbsFS := localizationFrame - planningFrame - camera` and,
Let `RelFS := planningFrame - camera`

--------------------------------------------------------------------------------------------------------------

`solveFrame.go` wants to be fed in `RelFS`. The "why" it wants `RelFS` is the result of `anyNonzero` and `anyPTG` being true should we choose to pass in `AbsFS`.

Once we are done with constructing our solverFrame, we then proceed to `PlanSingleWaypoint` and then into `planRelativeWaypoint`. From there, we enter `plannerSetupFromMoveRequest`.

We note that: One of the goals of the body of work is the allow placing of robot geometries exclusively through a FrameSystem and its corresponding seedMap.

This is where things get tricky. We have simplified the code to become
```
// create robot collision entities
movingGeometriesInFrame, err := pm.frame.Geometries(frameInputs)
```
Where `pm.Frame` is actually the `solverFrame` entity we have created upstream.
If we step into the `solverFrame.Geometries` method we iterate over `sf.movingFS` which was constructed using the `RelFS`. This will end up incorrectly placing the robot collision entities.

Hence, solverFrame code wants `RelFS` while plannerSetupFromMoveRequest code wants `AbsFS`.
For this reason, I say that
>We may want to move
>```
>	// re-root the frame system on the relative frame
>	relativeOnlyFS, err := pm.frame.fss.FrameSystemSubset(request.Frame)
>```
>upstream of this so that this is a non-issue.

is not the solution, given what we want to accomplish.

I am certain that there is a clean solution to the issue I have described above, I just need to think on it a little longer.
Would you agree with my problem description?

Reply:
fleshing out the details now but I think the solution here is that we will want to combine the 
LocalizationFrame (poseFrame) with the PlanningFrame(ptgGroupFrame) into a singular frame called WrappedFrame. This will circumvent this issue and we will allows to un-comment this code.

Yesterday, I went down the wrong train of thought as I thought this is an issue from plan construction. Instead, this is an issue for interacting with CheckPlan.

Reply:
spoke with ray, he recommended I use a `referenceframe.Model` instead of building a one off `WrapperFrame`

Reply:
spoke with Peter, moving the any-non zero check into planRelativeWayPoint circumvents this issue and we can pass in `AbsFS := localizationFrame - planningFrame - camera` to solverFrame construction